### DISCUSSION TITLE
Don't send a notification, but see existing notifications

### DISCUSSION THREAD

**Discussion by Dronakurl** [hoodie/notify-rust#235] [Q&A]
I want to view the current notifications (XDG), not send any new ones.
Can anyone please point me in the right direction?

**Comment by hoodie** [comment]
what to mean with view? like get a list of currently visible observations? I don't know of any api for that unfortunately. Or did you find anything by now? if so we could think about adding it.

**Comment by Dronakurl** [comment]
Yes, that's what I meant. I only found this [xdg site](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.impl.portal.Notification.html) which also only explains how to send notifications

**Comment by hoodie** [comment]
Yes I think thats the extend of it. It is an RPC API to send them off. Once sent you can't tell what has been sent. Probably also a good thing, considering that you don't necessarily want your application's notifications to be read by 3rd party applications.
